src/test/java/edu/kis/vh/nursery/RyhmersJUnitTest.java

```diff
@@ -44,34 +46,34 @@ public void testIsFull() {
 	@Test
 	public void testPeekaboo() {
 		IntArrayStack ryhmer = new IntArrayStack();
-		final int EMPTY_STACK_VALUE = -1;
+		final int EMPTY_STACK_VALUE = 0;

```

Comment:
Original constant should be used (it is better to use a constant from stacks interface).